### DISCUSSION TITLE
Welcome to KeepMyPassword-Desktop Discussions!

### DISCUSSION THREAD

**Discussion by xEdziu** [xEdziu/KeepMyPassword-Desktop#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we
  build together 💪.

**Comment by kamiloox** [comment]
Sure, I will be waiting for announcements here on my pretty well-comfortable-greenish-armless office chair with wheels
